src/applications/simple-forms/21-10210/config/form.js

```diff
@@ -1,55 +1,310 @@
+import vetAddrInfo from '../pages/vetAddrInfo';
+import vetContInfo from '../pages/vetContInfo';
+import statement from '../pages/statement';
 
-// const { } = fullSchema.properties;
+// "Flows" in comments below map to "Stories" in the mockups:
+// https://www.sketch.com/s/a11421d3-c148-41a2-a34f-3d7821ea676f
 
-// const { } = fullSchema.definitions;
+// mock-data import for local development
+// import the appropriate file [...-flow?.json] for the flow you're working on, or
+// test-data-no-stmtinfo.json for all flows [select claimOwnership & claimantType via UI]
+import testData from '../tests/fixtures/data/test-data-no-stmtinfo.json';
 
+const mockData = testData.data;
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: '/v0/api',
+  submitUrl: `${environment.API_URL}/forms_api/v1/simple_forms`,
   submit: () =>
     Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
   trackingPrefix: 'lay-witness-10210-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
   formId: '21-10210',
   saveInProgress: {
-    // messages: {
-    //   inProgress: 'Your claims application (21-10210) is in progress.',
-    //   expired: 'Your saved claims application (21-10210) has expired. If you want to apply for claims, please start a new application.',
-    //   saved: 'Your claims application has been saved.',
-    // },
+    messages: {
+      inProgress: 'Your claims application (21-10210) is in progress.',
+      expired:
+        'Your saved claims application (21-10210) has expired. If you want to apply for claims, please start a new application.',
+      saved: 'Your claims application has been saved.',
+    },
   },
   version: 0,
-  prefillEnabled: false,
+  prefillEnabled: true,
   savedFormMessages: {
-    notFound: 'Please start over to apply for claims.',
-    noAuth: 'Please sign in again to continue your application for claims.',
+    // notFound: 'Please start over to apply for claims.',
+    // noAuth: 'Please sign in again to continue your application for claims.',
+  },
+  title: 'Submit a Lay/Witness Statement',
+  subTitle: 'Equal to submitting a Lay/Witness Statement (VA Form 21-10210)',
+  defaultDefinitions: {
+    privacyAgreementAccepted: {
+      type: 'boolean',
+      enum: [true],
+    },
   },
-  title: '21-10210 Lay/Witness Statement',
-  defaultDefinitions: {},
   chapters: {
-    chapter1: {
-      title: 'Chapter 1',
+    statementInfoChapter: {
+      // for ALL Flows
+      // claimOwnership & claimantType decide which Flow to render downstream
+      title: 'Who is submitting this statement?',
+      hideFormNavProgress: true,
+      pages: {
+        claimOwnershipPage: {
+          path: 'claim-ownership',
+          title: 'Who is submitting this statement?',
+          // we want req'd fields prefilled for testing/previewing
+          // one single initialData prop here will suffice for entire form
+          initialData:
```

Comment:
will we want to remove this line when the form goes live? but I agree it seems helpful for testing.

Reply:
@rhasselle-oddball , confirming here that my conditional initialData does work.  I double-checked Staging and the fields are empty.  :)